README.md

```diff
@@ -51,5 +50,42 @@ cat my_awesome_file.txt
 Command | Explanation
 --- | ---
 `cross-cat <filename>` | For example, in case of PowerShell, ensure that this module is run instead of `cat` built-in alias to `Get-Content`
-`npx cross-cat <filename>` | To run without installation, although npm information would be displayed as well
-`cat --help` | To get all the available options
\ No newline at end of file
+`npx cross-cat <filename>` | To run without installation
+`cross-cat --help` | To get all the available options
+
+### Escaping spaces in `package.json` scripts
+
+```json
+"scripts": {
+	"show:awesome": "cross-cat \"/awesome folder/awesome filename\""
+}
+```
+
+### Spaces and npx
+
+Currently runs **via npx** of **locally** installed package cannot process filenames with spaces correctly due to npm behavior. Workaround:
+
+```bash
+node node_modules/cross-cat/src/cross-cat.js <filename>
+```
+
+#### Spaces support
+
+`+`
+: Working
+
+`=`
+: Non applicable
+
+`-`
+: Not working (workaround above)
+
+Command used | Global | Local | Not installed
+--- | --- | --- | ---|
+cat | + | = | =
+cat (from script) | + | + | =
+cross-cat | + | = | =
+cross-cat (from script) | + | + | =
+npx cross-cat | - | - | +

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-cell-padding] Cell should be compact](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5222203)